studio/pages/new/[slug].tsx

```diff
@@ -254,131 +264,161 @@ export const Wizard = observer(() => {
+                      value={v}
+                      addOnBefore={() => <IconDollarSign />}
                     >
-                      {passwordStrengthMessage
-                        ? passwordStrengthMessage
-                        : 'This is the password to your postgres database, so it must be a strong password and hard to guess.'}
-                    </span>
-                  </>
-                }
-                errorMessage={
-                  passwordStrengthWarning
-                    ? `${passwordStrengthWarning}. ${passwordErrorMessage}.`
-                    : passwordErrorMessage
-                }
-              />
-            </Panel.Content>
-
-            <Panel.Content className="Form section-block--body has-inputs-centered border-b border-panel-border-interior-light dark:border-panel-border-interior-dark">
-              <FormField
-                // @ts-ignore
-                label="Region"
-                type="select"
-                choices={REGIONS}
-                value={dbRegion}
-                onChange={onDbRegionChange}
-                description="Select a region close to you for the best performance."
-              />
-            </Panel.Content>
-
-            <Panel.Content className="Form section-block--body has-inputs-centered ">
-              <Listbox
-                label="Pricing Plan"
-                layout="horizontal"
-                value={dbPricingPlan}
-                onChange={onDbPricingPlanChange}
-                // @ts-ignore
-                descriptionText={
-                  <>
-                    Select a plan that suits your needs.&nbsp;
-                    <a className="underline" target="_blank" href="https://supabase.com/pricing">
-                      More details
-                    </a>
-                  </>
-                }
-              >
-                {Object.entries(PRICING_PLANS).map(([k, v]) => (
-                  <Listbox.Option
-                    key={k}
-                    label={v}
-                    value={v}
-                    addOnBefore={() => <IconDollarSign />}
-                  >
-                    {`${v}${
-                      v === PRICING_PLANS.PRO
-                        ? ' - $25/month'
-                        : v === PRICING_PLANS.PAYG
-                        ? ' - $25/month plus usage costs'
-                        : ''
-                    }`}
-                  </Listbox.Option>
-                ))}
-              </Listbox>
-
-              {!isSelectFreeTier && isOverFreeProjectLimit && <FreeProjectLimitWarning />}
-              {!isSelectFreeTier && isEmptyPaymentMethod && (
-                <EmptyPaymentMethodWarning stripeCustomerId={stripeCustomerId} />
-              )}
-            </Panel.Content>
-          </>
+                      {`${v}${
+                        v === PRICING_PLANS.PRO
+                          ? ' - $25/month'
+                          : v === PRICING_PLANS.PAYG
```

Comment:
>maybe do that in another PR then, since its not even supported in the org/project billing pages yet

we should remove PAYG from PRICING_PLANS to be consistent with org/project billing pages

Reply:
gotcha! removed